api/whatWeDo.js

```diff
@@ -48,6 +55,7 @@ export async function fetchApproachDePage(locale) {
   const content = await fetchSingleEntry('pageApproachDE', locale)
   return {
     ...content,
+    teamMember: unwrapTeamMember(content && content.teamMember),
```

Comment:
The `content &&` is redundant here. The `...content` one line above will crash anyway if `content` is `undefined`.

Reply:
Good catch. I will try to check it before the spread.